copyright.txt

```diff
@@ -0,0 +1,4 @@
+#
+# Copyright (C) CloudTruth, Inc.
```

Comment:
This is missing a year, which I think we want to have https://termly.io/resources/articles/copyright-examples/

Reply:
Initially I was thinking we couldn't have a year because nothing would update it. Then I remembered humans.